### PR TITLE
Translation of "Hooks at a Glance" page

### PR DESCRIPTION
**Pull request by GiuMagnani**
https://github.com/reactjs/es.reactjs.org/issues/4#issuecomment-460337062

### CODE REVIEW

File: content/docs/hooks-overview.md [hunk 1]
```diff
@@ -38,53 +39,54 @@ function Example() {
...
 
-#### But what is a Hook?
+#### Pero qué es un Hook?
 
-Hooks are functions that let you “hook into” React state and lifecycle features from function components. Hooks don't work inside classes -- they let you use React without classes. (We [don't recommend](/docs/hooks-intro.html#gradual-adoption-strategy) rewriting your existing components overnight but you can start using Hooks in the new ones if you'd like.)
+Hooks son funciones que te permiten "enganchar" el estado de React y el ciclo de vida desde componentes funcionales. Los ganchos no funcionan dentro de las clases -- te permiten usar React sin clases. ([No recomendamos](/docs/hooks-intro.html#gradual-adoption-strategy) reescribir tus componentes existentes de la noche a la mañana, pero puedes comenzar a usar Hooks en los nuevos si quieres.)
```
Comment (alejandronanez) [inline content/docs/hooks-overview.md]: Let's keep `hooks` as `hooks` for now :)

```suggestion
Hooks son funciones que te permiten "enganchar" el estado de React y el ciclo de vida desde componentes funcionales. Los hooks no funcionan dentro de las clases -- te permiten usar React sin clases. ([No recomendamos](/docs/hooks-intro.html#gradual-adoption-strategy) reescribir tus componentes existentes de la noche a la mañana, pero puedes comenzar a usar Hooks en los nuevos si quieres.)
```

File: content/docs/hooks-overview.md [hunk 2]
```diff
@@ -38,53 +39,54 @@ function Example() {
...
+React proporciona algunos Hooks incorporados como `useState`. También puedes crear tus propios Hooks para reutilizar el comportamiento con estado entre diferentes componentes. Primero veremos los Hooks incorporados.
 
->Detailed Explanation
+>Explicación Detallada
 >
->You can learn more about the State Hook on a dedicated page: [Using the State Hook](/docs/hooks-state.html).
+>Puedes aprender más sobre el Hook de Estado en la pagina dedicada: [Usando el Hook de Estado](/docs/hooks-state.html).
 
-## ⚡️ Effect Hook
+## ⚡️ Hook de Estado
```
Comment (alejandronanez) [inline content/docs/hooks-overview.md]:
```suggestion
## ⚡️ Hook de Efecto
```

File: content/docs/hooks-overview.md (hunk 1 above)
Comment (GiuMagnani) [reply content/docs/hooks-overview.md]: Missed that one, thank you!

File: content/docs/hooks-overview.md (hunk 2 above)
Comment (GiuMagnani) [reply content/docs/hooks-overview.md]: Thank you for that one!

**Comment by GiuMagnani** [comment]
Thanks @alejandronanez for the corrections! I already committed them. Let me know if there is anything else.

**Comment by GiuMagnani** [comment]
@alejandronanez I'll take a look, thanks!

**Pull request merged**